### DISCUSSION TITLE
creating separate checklist

### DISCUSSION THREAD

**Discussion by Chethandev12** [magicbell/community#138] [Questions]
Is it possible to add our checklist or tasks in Magicbell?   please refer to below image
![image](https://github.com/magicbell-io/public/assets/106078281/5e0415d7-45ff-487c-8ffc-cc94d71be5af)

**Comment by unamashana** [comment]
Please see https://github.com/orgs/magicbell-io/discussions/139#discussioncomment-7127044

Happy to answer any followup questions there.
